packages/commonwealth/server/routes/contractAbis/createContractAbi.ts

```diff
@@ -0,0 +1,103 @@
+import { NextFunction } from 'express';
+import Web3 from 'web3';
+import BN from 'bn.js';
+import { Op } from 'sequelize';
+import { factory, formatFilename } from 'common-common/src/logging';
+import { DB } from 'server/models';
+import { ContractAbiAttributes } from 'server/models/contract_abi';
+import { AbiItem } from 'web3-utils';
+import { parseAbiItemsFromABI } from 'commonwealth/client/scripts/helpers/abi_utils';
```

Comment:
Why are we importing client code on the server?! Should be common-common, or shared/.